addons/uom/data/uom_data.xml

```diff
@@ -39,28 +43,48 @@
             <field name="factor_inv" eval="12"/>
             <field name="uom_type">bigger</field>
         </record>
+
         <record id="product_uom_kgm" model="uom.uom">
             <field name="category_id" ref="product_uom_categ_kgm"/>
             <field name="name">kg</field>
             <field name="factor" eval="1"/>
             <field name="rounding" eval="0.001"/>
+            <field name="international_unit">kilogram</field>
         </record>
         <record id="product_uom_gram" model="uom.uom">
             <field name="category_id" ref="product_uom_categ_kgm"/>
             <field name="name">g</field>
             <field name="factor" eval="1000"/>
             <field name="uom_type">smaller</field>
+            <field name="international_unit">gram</field>
+        </record>
+
+        <record id="product_uom_time_day" model="uom.uom">
```

Comment:
Why product_uom_time and not time_uom or uom_time ?
just because of the naming of other xml ids? :D

Reply:
Yes, but it is true: 'uom_time_xxx' is better !